src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -1598,12 +1603,58 @@ def process_gallery_image_version_namespace(cmd, namespace):
                     raise CLIError("usage error: {} is an invalid target region argument. "
                                    "The second part must be a valid integer replica count.".format(t))
 
+            # Parse target region encryption, example: ['des1,0,des2,1,des3', 'null', 'des4']
+            encryption = None
+            if hasattr(namespace, 'target_region_encryption') and namespace.target_region_encryption:
```

Comment:
what if these cases?

- ['null,0,des2,1,des3']
- ['']

Reply:
os des == null is handled.
[''] is invalid, good question, should CLI handle wrong data